pkg/integrations/v4/dm/emitter.go

```diff
@@ -17,6 +18,7 @@ import (
 	"github.com/newrelic/infrastructure-agent/pkg/integrations/legacy"
 	"github.com/newrelic/infrastructure-agent/pkg/integrations/v4/protocol"
 	"github.com/newrelic/infrastructure-agent/pkg/log"
+	"github.com/tevino/abool"
```

Comment:
just a thought from security paranoid, do we add new libraries from small developers without any sec review?

Reply:
vendor dependencies get scanned by Snyk at CI